allensdk/brain_observatory/behavior/data_objects/trials/trial.py

```diff
@@ -0,0 +1,423 @@
+from typing import List, Dict
```

Comment:
It looks like most of the methods in this file need return type hints, if possible